modules/installation-azure-permissions.adoc

```diff
@@ -11,4 +12,4 @@
 * `User Access Administrator`
 * `Contributor`
 
-To set roles on the Azure portal, see the link:https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal[Manage access to Azure resources using RBAC and the Azure portal] in the Azure documentation.
\ No newline at end of file
+To set roles on the Azure portal, see the link:https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal[Manage access to Azure resources using RBAC and the Azure portal] in the Azure documentation.
```

Comment:
It just shifted to line 15 because I added the comment on line 5 
https://github.com/openshift/openshift-docs/pull/63336/files#diff-3aaa8dbcdb1a5c6a7177a429a8e1501b945938712e860d56eeb1070b7d50b48eR5